GETTING_STARTED.md

```diff
@@ -45,6 +48,9 @@ This app requires access to a Postgres database to store resource values. Heroku
 1. Navigate to `Resources` -> `Add-ons`
 1. Search for `postgres` and attach a free "Hobby Dev" account
 
+<br/>
+![](docs/app_postgres.png)
```

Comment:
These are not rendering correctly from what I can tell.  Could you remove the <br /> ? or add an additional empty line?

When viewing on your fork: https://github.com/cvasilak/workshop-web-app/blob/doc/GETTING_STARTED.md

![Screenshot 2019-11-28 at 18 09 38](https://user-images.githubusercontent.com/20280797/69826236-454f0180-120a-11ea-85ec-bd17cf25b5e2.png)